### PR TITLE
Add property grid widget via extension

### PR DESCRIPTION
**Pull request by bsteinbk**
Update ui-test-extension to provide a basic property widget.

Update map-layers so when map-layers is used as an extension default props are passed to the widget.

### CODE REVIEW

**Comment by aruniverse** [comment]
Just adding myself, to keep track of the ui-test app changes. thx again for adding that sample Bill!

**Review by DanEastBentley — APPROVED** [review]

File: extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx [hunk 1]
```diff
@@ -21,14 +21,21 @@ export class MapLayersUiItemsProvider implements UiItemsProvider {
 
   public provideWidgets(_stageId: string, stageUsage: string, location: StagePanelLocation, section: StagePanelSection | undefined): ReadonlyArray<AbstractWidgetProps> {
     const widgets: AbstractWidgetProps[] = [];
+    const mapLayerOptions: MapLayerOptions = {
+      hideExternalMapLayers: false,
+      mapTypeOptions: { supportTileUrl: true, supportWmsAuthentication: true },
```
Comment (mdastous-bentley) [inline extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx:26]: I would leave supportTileUrl to false

File: extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx [hunk 2]
```diff
@@ -21,14 +21,21 @@ export class MapLayersUiItemsProvider implements UiItemsProvider {
 
   public provideWidgets(_stageId: string, stageUsage: string, location: StagePanelLocation, section: StagePanelSection | undefined): ReadonlyArray<AbstractWidgetProps> {
     const widgets: AbstractWidgetProps[] = [];
+    const mapLayerOptions: MapLayerOptions = {
+      hideExternalMapLayers: false,
+      mapTypeOptions: { supportTileUrl: true, supportWmsAuthentication: true },
+      fetchPublicMapLayerSources: true,
```
Comment (mdastous-bentley) [inline extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx:27]: I would leave this to one false too.

**Review by mdastous-bentley — APPROVED** [review]

File: extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx (hunk 2 above)
Comment (bsteinbk) [reply extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx]: done

File: extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx (hunk 1 above)
Comment (bsteinbk) [reply extensions/map-layers/src/ui/MapLayersUiItemsProvider.tsx]: done

**Pull request merged**